code/modules/planar_lighting/light_effect_cast.dm

```diff
@@ -12,11 +12,15 @@
 
 	// Clear overlays, blank slate.
 	overlays.Cut()
```

Comment:
Could likely save even more by removing this and assigning `list()` if the proc needs to return early.